python/pip_install/pip_repository_requirements.bzl.tmpl

```diff
@@ -34,6 +34,21 @@ def dist_info_requirement(name):
 def entry_point(pkg, script = None):
     if not script:
         script = pkg
+
+    # buildifier: disable=print
+    print("""
+DEPRECATED: please replace this instance of entry_point with the following:
```

Comment:
Please include a link to the docs, and mention that you must put this rule into a project's BUILD file.